### DISCUSSION TITLE
The round algorithm of Altshuler et al. 2017

### DISCUSSION THREAD

**Discussion by shlomihod** [PythonOT/POT#322] [Q&A]
Is the ROUND algorithm (second in the paper) of Altshuler et al. 2017 implemented? I couldn't find it.
If not, I'd like to add it - what would be the best way to do so?

**Comment by rflamary** [comment]
The algorithm is indeed not implemented if I remember well but we want it ;)

If you want to add it, maybe we can add it as an optional post processing (if I remember well it is done only at the end) of the sinkhorn methods? This means that you need to change the sinkhorn function and all the variants of algorithms for sinkhorn.

Or this can be done as a unique function thata takes the log (dual variables) of sinkhorn and return the round solution?

In any case you will need to do a Pull request and to design unit tests to check
